packages/shopify-codemod/transforms/split-assignment-sequences.js

```diff
@@ -0,0 +1,29 @@
+export default function splitAssignmentSequences({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+  return j(source)
+    .find(j.ExpressionStatement, {
+      expression: (node) => j.SequenceExpression.check(node),
+    })
+    .filter((path) => {
+      const expressions = path.value.expression.expressions;
+      const lastExpression = expressions[expressions.length - 1];
+      const otherExpressions = expressions.slice(0, -1);
+
+      return j.Identifier.check(lastExpression) &&
+        otherExpressions.every(j.AssignmentExpression.check) &&
+        otherExpressions.every((expression) =>
+          expression.right.object.name === lastExpression.name,
+        );
+    })
+    .replaceWith((path) => {
```

Comment:
`forEach` and call `replace(null)` at the end? It feels awkward to prepend in `replaceWith`.